### PR TITLE
Grunt.Grunt version 20.23.2

### PR DESCRIPTION
**Pull request by hgroendal**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: codeflow:open?pullrequest=https://github.com/microsoft/winget-pkgs/pull/110148&drop=dogfoodAlpha

### CODE REVIEW

**Comment by hgroendal** [comment]
> Hello @hgroendal,
> 
> One or more of the installer URLs doesn't appear valid.
> 
> This may happen for sites with policies prohibiting distribution or use by third parties.
> 
> This may happen for URLs pointing to domains that do not align with the publisher domain or package domain. If you could provide supporting evidence from the publisher that the URLs for the installer are correct, that would help us to validate and approve this PR.
> 
> Template: msftbot/validationError/urls/domain

> msftbot/validationError/urls/domain

The domain for the application download is "app.grunt.pro" and should be correct based on the names of the developers of the application.

File: manifests/g/Grunt/Grunt/20.23.2/Grunt.Grunt.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.6.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: Grunt.Grunt
+PackageVersion: 20.23.2
+PackageLocale: en-US
+Publisher: Grunt
+PackageName: Grunt
```
Comment (stephengillie) [inline manifests/g/Grunt/Grunt/20.23.2/Grunt.Grunt.locale.en-US.yaml:8]:
```suggestion
PrivacyUrl:   https://app.grunt.pro/legal
PackageName: Grunt
PackageUrl: https://www.grunt.pro/
```

**Comment by stephengillie** [comment]
Hi @hgroendal,

This package's ISV appears to require registration before downloading. Unfortunately, this package might not be a good fit for inclusion into the WinGet public manifests. Please consider using a local manifest (WinGet install --manifest C:\path\to\manifest\files\) for local installations.

**Comment by hgroendal** [comment]
> Hi @hgroendal,
> 
> This package's ISV appears to require registration before downloading. Unfortunately, this package might not be a good fit for inclusion into the WinGet public manifests. Please consider using a local manifest (WinGet install --manifest C:\path\to\manifest\files) for local installations.

Hi, 

What do you mean by ISV? And where is it that you have to run a registration before downloading? Me and several colleagues were able to download the package just fine, without having to log on. 

Please see the attached link for the msi download: https://app.grunt.pro/download.

**Comment by stephengillie** [comment]
> Hi,
> 
> What do you mean by ISV? 

Sorry, Independent Software Vendor. This umbrella term groups individual developers, small teams, large teams, and multinational corporations into parties which release software. We have requirements around this, including (for example) that the package has to come from the ISV's hosting in one way or another, as opposed to a third-party mirror. 

> And where is it that you have to run a registration before downloading? Me and several colleagues were able to download the package just fine, without having to log on.

Another requirement is generally to be able to navigate from the PackageUrl to the InstallerUrl. Again, apologies for not specifying this before. I found a "Download" link on the ISV's site, but this led to a registration page. 

> Please see the attached link for the msi download: https://app.grunt.pro/download.

This should be the PackageUrl.

**Comment by hgroendal** [comment]
> The domain for the application download is "app.grunt.pro" and should be correct based on the names of the developers of the application.



> Hello @hgroendal,
> 
> One or more of the installer URLs doesn't appear valid.
> 
> This may happen for sites with policies prohibiting distribution or use by third parties.
> 
> This may happen for URLs pointing to domains that do not align with the publisher domain or package domain. If you could provide supporting evidence from the publisher that the URLs for the installer are correct, that would help us to validate and approve this PR.
> 
> Template: msftbot/validationError/urls/domain

The domain for the application download is "app.grunt.pro" and should be correct based on the names of the developers of the application. The EULA also states that you are allowed to distribute the application as long as you don't change the contents or give it to third parties: https://app.grunt.pro/terms.

**Comment by stephengillie** [comment]
Does this package have a dependency on VisualStudio Tools? 
image

**Comment by hgroendal** [comment]
> Does this package have a dependency on VisualStudio Tools? image

Hi. We have published this application to our end users before, without including Visual Studio Runtime before, but seeing as it is an Office addin, it might need this as a prerequisite.

**Comment by stephengillie** [comment]
> Does this package have a dependency on VisualStudio Tools? image

This is unchanged.

**Comment by stephengillie** [comment]
[This page](https://learn.microsoft.com/en-us/visualstudio/vsto/assemblies-in-the-visual-studio-tools-for-office-runtime?view=vs-2022) suggests that the missing assembly might be supplied by the `Microsoft.DotNet.Framework.DeveloperPack_4` package.

**Review by stephengillie — APPROVED** [review]

**Pull request merged**